subconscious/model.py

```diff
@@ -314,17 +326,51 @@ def get_index_key(cls, column_name):
                 first_iteration = False
             else:
                 result_set = result_set.intersection(temp_set)
+            if order_by:
+                pairs = []
+
+                def _pair_with_zero_score(x):
+                    return [0, x]
+
+                [pairs.extend(_pair_with_zero_score(x)) for x in result_set]
+                if pairs:
+                    filtered_res_key = 'filtered_result-{}'.format(uuid.uuid1())
+                    await db.zadd(filtered_res_key, pairs[0], pairs[1], *pairs[2:])
```

Comment:
Looks like because of the *pairs in the zadd api , I don't think we can use named arguments here. Whatever I've tried is resulting in syntax error!